src/BuildingBlocks/EventBus/EventBusServiceBus/DefaultServiceBusPersisterConnection.cs

```diff
@@ -1,58 +1,50 @@
-﻿using Microsoft.Azure.ServiceBus;
+﻿using Azure.Messaging.ServiceBus;
+using Azure.Messaging.ServiceBus.Administration;
 using System;
 
 namespace Microsoft.eShopOnContainers.BuildingBlocks.EventBusServiceBus
 {
     public class DefaultServiceBusPersisterConnection : IServiceBusPersisterConnection
     {
-        private readonly ServiceBusConnectionStringBuilder _serviceBusConnectionStringBuilder;
-        private readonly string _subscriptionClientName;
-        private SubscriptionClient _subscriptionClient;
-        private ITopicClient _topicClient;
+        private readonly string _serviceBusConnectionString;
+        private ServiceBusAdministrationClient _subscriptionClient;
+        private ServiceBusClient _topicClient;
```

Comment:
The equivalent of the TopicClient would be ServiceBusSender.

Reply:
In the `EventBusServiceBus.cs` file, `TopicClient` was used to create processor, so I think it need to be `ServiceBusClient` here.